WalletWasabi.Fluent/Models/Transactions/PrivacySuggestionsModel.cs

```diff
@@ -231,10 +236,17 @@ private async Task<List<ChangeAvoidanceSuggestion>> CreateChangeAvoidanceSuggest
 		// Only allow to create 1 more input with BnB. This accounts for the change created.
 		int maxInputCount = transaction.SpentCoins.Count() + 1;
 
+		var cjManager = Services.HostedServices.Get<CoinJoinManager>();
```

Comment:
Consider getting CJM in the ctor, as this is the second place where you reference it.